toke.c

```diff
@@ -12731,12 +12731,12 @@ look something like this:
     static OP *my_keyword_plugin(pTHX_
         char *keyword_plugin, STRLEN keyword_len, OP **op_ptr)
```

Comment:
I think it would be better to fix the name of the argument to be `keyword_ptr`, since it's a pointer to the keyword being parsed, not the plugin.

Reply:
Good point! Adjusted the commit.